framework/data/Sort.php

```diff
@@ -301,9 +308,7 @@ public function setAttributeOrders($attributeOrders, $validate = true)
      */
     public function getAttributeOrder($attribute)
     {
-        $orders = $this->getAttributeOrders();
-
-        return isset($orders[$attribute]) ? $orders[$attribute] : null;
+        return ArrayHelper::getValue($this->getAttributeOrders(), $attribute);
```

Comment:
this adds too much overhead, it does not make sense to replace a simple isset() with the complex implementation that `ArrayHelper::getValue()` has.